test/fixtures/mson/enum-variants.json

```diff
@@ -1634,70 +1634,6 @@
         }
       ]
     },
-    {
-      "element": "annotation",
-      "meta": {
-        "classes": {
-          "element": "array",
-          "content": [
-            {
-              "element": "string",
-              "content": "warning"
-            }
-          ]
-        }
-      },
-      "attributes": {
-        "code": {
-          "element": "number",
-          "content": 4
-        },
-        "sourceMap": {
-          "element": "array",
-          "content": [
-            {
-              "element": "sourceMap",
-              "content": [
-                {
-                  "element": "array",
-                  "content": [
-                    {
-                      "element": "number",
-                      "attributes": {
-                        "line": {
-                          "element": "number",
-                          "content": 9
-                        },
-                        "column": {
-                          "element": "number",
-                          "content": 3
-                        }
-                      },
-                      "content": 145
-                    },
-                    {
-                      "element": "number",
-                      "attributes": {
-                        "line": {
-                          "element": "number",
-                          "content": 9
-                        },
-                        "column": {
-                          "element": "number",
-                          "content": 74
-                        }
-                      },
-                      "content": 72
-                    }
-                  ]
-                }
-              ]
-            }
-          ]
-        }
-      },
-      "content": "duplicit value in enumeration"
```

Comment:
I think this warning is correct and shouldn't be removed. The only thing that needed changing is that the "mson example value" of an `enum` is added to `enumerations` _after_ all it's nested members are. What do you think?

Reply:
Although, perhaps it makes better sense to give a warning in the exact opposite example, where we introduce a new type to `enumerations` because the example does not match any nested type.